docs/content/docs/dev/configuration/maven.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
I'm leaning towards moving the content of Maven, Gradle and SBT all to sections under the Overview. Reasons for that is that if something changes on the Project Configuration, you can immediately change it in one go and keep things in sync. Probably with the tabs option to see different setups (a tab for a Maven example, tab for Gradle etc). Something like that?

What made you think about splitting them to separate pages?

Reply:
In the first iteration, I thought that it made the page too long and that too much scrolling makes for bad UX, but I can try the tabs idea.

Reply:
That's a fair point but I'm also not sure if we provide perfect support for both Gradle and SBT. I would lean towards integrating those in one page.

Reply:
@MartijnVisser you mean something like this? https://github.com/apache/flink/pull/18353#discussion_r789423489